hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -182,6 +185,14 @@ public MapConfigReadOnly getAsReadOnly() {
         return readOnly;
     }
 
+    public boolean isHotRestartEnabled() {
+        return hotRestartEnabled;
+    }
+
+    public void setHotRestartEnabled(boolean hotRestartEnabled) {
```

Comment:
I think you should add a javadoc here, explaining what `hotRestartEnabled` means.